students/jvirtue/lesson09/mailroom.py

```diff
@@ -0,0 +1,154 @@
+#Lesson 9 Assignment 1
+#Mailroom Part 5 Assignment
+#Jason Virtue 03/13/2019
+#UW Self Paced Python Course
+
+class Donor():
+    def __init__(self, name):
+        self._name = name
+        self._donations = []
+
+    @property
+    def name(self):
+        return self._name
+
+    @name.setter
+    def name(self,donor_name):
+        self._name = donor_name
+
+    @property
+    def donations(self):
+        return self._donations
+    
+    #Add Donations from Dict
+    def amount_add(self,donor_donation):
+        self.donations.append(float(donor_donation))
+
+    #Thank you message in console app
+    def thank_you(self, name, donation):
+        return "Thank you {} ".format(name) + "for the donation in the amount of {}!".format(donation)
+
+class Donor_Collections():
+    def __init__(self):
+        self._donors = []
+    
+    @property
+    def donors(self):
+        return self._donors
+
+    def donor_add(self,donor):
+        self.donors.append(donor)
+
+    #List out unique donors
+    def donor_list(self):
+        name_list = []
+        for donor in self._donors:
+            name_list.append(donor.name)
+        return name_list
```

Comment:
great place for a comprehension 😉 

`return [donor.name for donor in self._donors]`